src/app/app-extras.module.ts

```diff
@@ -1,5 +1,4 @@
 import { NgModule } from '@angular/core';
-
```

Comment:
I think we want this blank line back.  Separates our local imports from the node_modules.